tests/Cache/MemoryCacheItemPoolTest.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Auth\Tests;
+
+use Google\Auth\Cache\Item;
+use Google\Auth\Cache\MemoryCacheItemPool;
+use Psr\Cache\InvalidArgumentException;
+
+class MemoryCacheItemPoolTest extends \PHPUnit_Framework_TestCase
+{
+    private $pool;
+
+    public function setUp()
+    {
+        $this->pool = new MemoryCacheItemPool();
+    }
+
+    public function saveItem($key, $value)
+    {
+        $item = $this->pool->getItem($key);
+        $item->set($value);
+        $this->assertTrue($this->pool->save($item));
+
+        return $item;
+    }
+
+    public function testGetsFreshItem()
+    {
+        $item = $this->pool->getItem('item');
+        
+        $this->assertInstanceOf(Item::class, $item);
```

Comment:
We need to change this, or skip in PHP 5.4. [Travis is failing](https://travis-ci.org/google/google-auth-library-php/jobs/147305702#L244)